tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1479,4 +1479,47 @@ class CompletionSuite extends BaseCompletionSuite {
     "",
   )
 
+  check(
+    "pkg".tag(IgnoreScalaVersion.for3LessThan("3.1.3")),
+    s"""|object Foo {
+        |  scala.coll@@
+        |}
+        |""".stripMargin,
+    "collection scala",
+  )
+
+  check(
+    "pkg".tag(IgnoreScalaVersion.for3LessThan("3.1.3")),
```

Comment:
```suggestion
    "pkg-type".tag(IgnoreScalaVersion.for3LessThan("3.1.3")),
```

Reply:
so that the three tests would not have the same name

Reply:
Fixed!